pkg/agent/proxy/utils.go

```diff
@@ -0,0 +1,40 @@
+package proxy
```

Comment:
I feel not very necessary to add a new file just for one func, but ok with it too if you prefer.

Reply:
Moved it to proxier.go